### DISCUSSION TITLE
Does Wix 4.x  support  cli experience ？

### DISCUSSION THREAD

**Discussion by John0King** [wixtoolset/issues#6613] [Questions]
now there is  `winget` ,  and  also  the nano-windows that do not have a UI,  does the installer that created by wix support both UI and CLI experience ?

-------------------
and I'm new to Wix ,  where can I lean how to use wix to create my installer ?

**Comment by robmen** [comment]
Yes.

**Reply by John0King** [reply]
@robmen   I means that not `wix` it self, but the executable installer that created by `wix`.

a few days before I see docker's installer is act like a console app with in `winget` , but double click is a GUI app

**Reply by John0King** [reply]
@robmen   I means that not `wix` it self, but the executable installer that created by `wix`.

a few days before I see docker's installer is act like a console app with in `winget` , but double click is a GUI app
